.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,15 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: true  # default
+contact_links:
+- name: '💬 IRC: #pypa @ Freenode'
+  url: https://webchat.freenode.net/#pypa
+  about: Chat with devs.
+- name: 🤷💻🤦 Discourse
+  url: https://discuss.python.org/c/packaging
+  about: Please ask typical Q&A here.
+- name: 🔐 Security Policy
+  url: https://pypi.org/security/
```

Comment:
I don't think the guidance for reporting PyPI/Warehouse security issues will be the same as for pip, at least because we haven't OK'd using that email distribution list for pip.

Reply:
Oh, I probably copy-pasted this when merging things from a few other configs that I contributed in different projects... So this should be removed for now.

But for the future, it'd be a good idea to formalize some security issues reporting guide and put it @ https://github.com/pypa/pip/security/policy and probably re-add this section here, plus docs. In a separate PR/discussion, of course.

cc @pradyunsg ^